requirements/local.in

```diff
@@ -0,0 +1,8 @@
+# Packages required for local development
+-r test.txt
+-r docs.txt
+
+django-debug-toolbar
+docker-compose==1.9.0               # lock to this version to workaround https://github.com/ansible/ansible/issues/20492
```

Comment:
actually don't think we need to do this locally. we only have to do it in production because the version of ansible that we are running is old, but we don't use ansible locally so we're fine to remove this pinning from local.in but leave it in production.in